src/drawers/rectDrawer.ts

```diff
@@ -84,16 +84,16 @@ export module Drawers {
       this._labelsTooWide = labelTooWide.some((d: boolean) => d);
     }
 
-    public draw(data: any[], drawSteps: DrawStep[], userMetadata: any) {
+    public draw(data: any[], drawSteps: DrawStep[]) {
```

Comment:
I wouldn't say broken... just majorly incorrect...
Yeah...
(The main of the problem is that we don't have a hard type on `metadata`, thus delegating to `any`)